packages/blocks/src/components/blocks/DruxtBlockBlockContent.vue

```diff
@@ -52,7 +52,7 @@ export default {
      *
      * @type {object}
      *
-     * @see {@link https://druxt.github.io/druxt-entity/api/components/DruxtEntity.html|DruxtEntity}
```

Comment:
These absolute URLs no longer exist and will need to be updated to relative links, however I can update them after I merge this in.